arangod/Aql/TraversalExecutor.h

```diff
@@ -180,7 +180,7 @@ class TraversalExecutorInfos {
       TraversalExecutorInfosHelper::OutputName type) const;
 
  private:
-  std::unique_ptr<traverser::Traverser>
+  std::shared_ptr<traverser::Traverser>
```

Comment:
Why is this made a shared_ptr now?
We only hand out a raw pointer, which we can from a unique_ptr as well.
I think this specific change should be rolled back.

Reply:
put back to unique ptr